packages/video_player/video_player_android/example/android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">www.sample-videos.com</domain>
+        <domain includeSubdomains="true">184.72.239.149</domain>
```

Comment:
I don't think we're supposed to hit external servers in tests.

Reply:
(I suspect that this already existed but the git diff history is weird for it and it is showing up as new code?)

Reply:
We have a bunch of tests that hit external servers. We've just started fixing them, but haven't gotten to video_player yet; I don't want to mix that into this PR.

It's showing as new code because video_player/video_player/example/android (which is what this is a copy of) still exists, so it's not a move. There's no concept of copies in git, unfortunately.